DesignSafe-CI/portal
# task/DES-2938 & DES-2941: App form updates

## Overview: ##
Adds user_guide_link to app entry model + uses it in app form header (for tapis + html apps)

## PR Status: ##


## Related Jira tickets: ##

* [DES-2941](https://tacc-main.atlassian.net/browse/DES-2941)
* [DES-2938](https://tacc-main.atlassian.net/browse/DES-2941)

## Summary of Changes: ##
- Add `user_guide_link` to app entry model + corresponding view
- Add 'userGuideLink' to portal app TS type to pick up above field + use it as first choice for user guide link in app form header
- Move app form header one level up to `AppViewLayout` to also cover HTML apps

## Testing Steps: ##
1. Go to https://designsafe.dev/rw/workspace/adcirc?version=55.01 and confirm there are no regressions in design
2. New field will be used in follow on CMS task 

## UI Photos:

## Notes: ##
